lib/progress_bar.rb

```diff
@@ -1,23 +1,30 @@
-
 require 'options'
 require 'highline'
 
 class ProgressBar
   Error = Class.new(StandardError)
   ArgumentError = Class.new(Error)
 
-  attr_accessor :count, :max, :meters
+  DefaultMeters = [:bar, :counter, :percentage, :elapsed, :eta, :rate]
```

Comment:
constants like this in ruby are `ALLCAPS_UNDERSCORE`

Reply:
Yeah ... I think I was keeping it consistent with `ArgumentError` above. Ill change it